common/types/offer_test.go

```diff
@@ -16,7 +16,7 @@ func TestOffer_MarshalJSON(t *testing.T) {
 
 	expected := fmt.Sprintf(`{
 		"version": "0.1.0",
-		"offerId": "%s",
+		"offerID": "%s",
```

Comment:
I actually think the original way is prettier, but I was shooting for consistency over beauty.